production/docker/config/loki-runtime.yaml

```diff
@@ -0,0 +1,11 @@
+overrides:
```

Comment:
I suspect this was used for your tests, but I'm skeptical about committing this to the actual repo

Reply:
indeed I used it for tests, and the reason why I wanted to commit it is so that anyone else wanting to test out Loki using docker-compose could do so easily with an already made config. Is this not the place to put such a thing?

Reply:
This setup is intended for production use, so I don't think these testing configs belong here.
I do like that you've setup alertmanager, so we can keep that, but the rest should be removed.

Reply:
I see. I will remove those configs then